simuleval/evaluator/scorers/quality_scorer.py

```diff
@@ -47,6 +47,42 @@ def add_sacrebleu_args(parser):
     )
 
 
+@register_quality_scorer("WER")
+class WERScorer(QualityScorer):
+    """
+    Compute Word Error Rate (WER)
+
+    Usage:
+        :code:`--quality-metrics WER`
+    """
+
+    def __init__(self, args) -> None:
+        super().__init__()
+        try:
+            import editdistance as ed
+        except ImportError:
+            raise ImportError("Please install editdistance to use WER scorer")
+        self.logger = logging.getLogger("simuleval.scorer.wer")
+        self.logger.warning("WER scorer only support language with spaces.")
+        self.logger.warning(
+            "Current WER scorer is on raw text (un-tokenized with punctuations)."
+        )
+        self.ed = ed
+
+    def __call__(self, instances: Dict) -> float:
+        distance = 0
+        ref_length = 0
+        for ins in instances.values():
+            distance += self.ed.eval(ins.prediction.split(), ins.reference.split())
+            ref_length += len(ins.reference.split())
+
+        return 100.0 * distance / ref_length if ref_length > 0 else 0
+
+    @classmethod
+    def from_args(cls, args):
+        return cls(args)
+
+
```

Comment:
Awesome! Thanks Xutai. I was just thinking of adding WER scorer to SimulEval for online ASR evaluations.